man/deepsea-stage-show.1

```diff
@@ -0,0 +1,69 @@
+.TH deepsea-stage-show 1 "DeepSea CLI man page"
+.SH NAME
+deepsea-stage-show - Shows DeepSea stage execution steps
+
+.SH SYNOPSIS
+deepsea stage show [--help] [--hide-state-steps] [--only-visible-steps]
+                   [--clear-cache] [--no-cache]
+                   <stage_name>
+
+.SH DESCRIPTION
+Shows a DeepSea stage steps ordered by execution order.
```

Comment:
Parses a DeepSea stage and generates a detailed report showing all steps in order of execution.